MIRROR.md

```diff
@@ -0,0 +1,24 @@
+# Mirroring CIRRUS-DAAC
+## Overview
+Process for creating a private version of CIRRUS-DAAC, while maintaining the ability
+to pull changes from CIRRUS-DAAC.
+## Prerequisites
+A new (private) repository to mirror CIRRUS-DAAC into. 
+
+## Mirroring a repo
+```bash
+cd to/your/project/directory
+git clone --bare https://github.com/asfadmin/CIRRUS-DAAC
+cd CIRRUS-DAAC.git
+git push --mirror https://github.com/account/new_repo
```

Comment:
This assumes `https://github.com/account/new_repo` exists, right? 

Can we add instructions for creating `new_repo`? I think this should have the details necessary: https://kbroman.org/github_tutorial/pages/init.html

Reply:
Sure!